app/models/work_paper.rb

```diff
@@ -275,6 +291,26 @@ def unzip_if_necesary
     end
   end
 
+  def sanitized_previous_code
+    @previous_code.sanitized_for_filename
+  end
+
+  def start_with_code file_name
+    code = sanitized_code
+    short_code = sanitized_code.sub(/(\w+_)\d(\d{2})$/, '\1\2')
+    condition = (file_name.start_with?(code, short_code) &&
```

Comment:
```suggestion
    result = (file_name.start_with?(code, short_code) &&
```

Y no hacen falta los paréntesis.

Reply:
Y de nuevo.